src/reconcile/rec_visibility.c

```diff
@@ -430,6 +431,7 @@ __timestamp_no_ts_fix(WT_SESSION_IMPL *session, WT_TIME_WINDOW *select_tw)
 
```

Comment:
line 426 should be
if (!WT_TIME_WINDOW_HAS_STOP_PREPARE && select_tw->stop_ts < select_tw->start_ts)

Reply:
done